pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -53,18 +46,28 @@ def solve_NLP_subproblem(solve_data, config):
     # | g(x) <= b  | -1    | g(x1) > b    | g(x1) - b            |
     # | g(x) >= b  | +1    | g(x1) >= b   | 0                    |
     # | g(x) >= b  | +1    | g(x1) < b    | b - g(x1)            |
-
+    flag = False
```

Comment:
The name of this variable does not say what it means (if `flag=True`, then there was an evaluation error in the constraints). May I suggest changing its name to something else? `evaluation_error` ?